CONTRIBUTING.md

```diff
@@ -95,14 +95,17 @@ Build as usual, and do `make clean testall` to test your contribution. If your c
 
 ##### General Formatting Guidelines For Julia code contributions
 
- - 4 space indent
+ - 4 spaces per indentation level, no tabs
  - use whitespace to make the code more readable
- - no whitespace at the end of a line
+ - no whitespace at the end of a line (trailing whitespace)
  - comments are good, especially when they explain the algorithm
+ - try to adhere to a 92 character line length limit
```

Comment:
Hard to beat the classics:

![latest](https://cloud.githubusercontent.com/assets/1732/5812107/80c6930a-a030-11e4-8fdd-2083417f33e4.png)

[ref](http://futurama.wikia.com/wiki/Olde_Fortran)